apps/ui-library/content/docs/nuxtjs/social-auth.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Social Authentication
+description: Social authentication block for Nuxt.js
+---
+
+<BlockPreview name="social-auth/auth/login" />
+
+<Callout className="mt-4">
+  The block is using Github provider by default, but can be easily switched by changing a single
+  parameter.
+</Callout>
+
+## Installation
+
+<BlockItem name="social-auth-nuxtjs" description="All needed components for the social auth flow" />
+
+## Folder structure
+
+This block assumes that you have already installed a Supabase client for Nuxt from the previous step.
+
+<RegistryBlock itemName="social-auth-nuxtjs" />
+
+## Usage
+
+Once you install the block in your Nuxt.js project, you'll get all the necessary pages and components to set up a social authentication flow.
+
+### Getting started
+
+After installing the block, you'll have the following environment variables in your `.env.local` file:
+
+```env
+NUXT_PUBLIC_SUPABASE_URL=
+NUXT_PUBLIC_SUPABASE_PUBLISHABLE_OR_ANON_KEY=
+```
+
+- If you're using supabase.com, you can find these values in the [Connect modal](https://supabase.com/dashboard/project/_?showConnect=true&connectTab=frameworks&framework=nuxtjs) under App Frameworks or in your project's [API settings](https://supabase.com/dashboard/project/_/settings/api).
```

Comment:
looks like this url needs a `using` param as well: 

https://supabase.com/dashboard/project/_?showConnect=true&connectTab=frameworks&framework=nuxt&using=supabasejs